### PR TITLE
test: Include cluster test

### PR DESCRIPTION
**Pull request by claudiachua**
Internal Ticket: [DI-1412](https://issues.corp.twilio.com/browse/DI-1412)

### CODE REVIEW

**Comment by claudiachua** [comment]
run pr validation

File: package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
     "dayjs": "^1.8.29",
     "https-proxy-agent": "^5.0.0",
     "jsonwebtoken": "^8.5.1",
+    "localtunnel": "2.0.2",
```
Comment (childish-sambino) [inline package.json:27]: This should be a `devDependency`.

File: cluster_test.spec.ts [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+        });
+      });
+
+      validationServer.listen(portNumber);
+      tunnel = await localtunnel({port: portNumber});
+    });
+
+    afterAll(() => {
+      tunnel.close();
+      testClient.studio.v2.flows(flowSid).remove();
```
Comment (childish-sambino) [inline cluster_test.spec.ts:98]: Won't this just delete the _last_ flow that was created?

File: package.json [hunk 3]
```diff
@@ -56,9 +57,10 @@
     "jshint": "jshint lib/rest/** lib/base/** lib/http/**",
     "jscs": "eslint lib/base/**/**.js lib/http/**/**.js --fix",
     "check": "npm run jshint && npm run jscs",
-    "ci": "npm run test && npm run nsp",
```
Comment (childish-sambino) [inline package.json:59]: I dislike the cluster tests being run on `make test` or `npm test`. These should _only_ be run when we're specifically targeting cluster tests.

File: cluster_test.spec.ts [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+
+    test("Should validate POST request", async () => {
+      await validateRequest('POST');
+      expect(validationCount).toBe(2);
+    })
+  });
+})
+
+test("Should list available numbers", () => {
+  return testClient.availablePhoneNumbers('US')
```
Comment (childish-sambino) [inline cluster_test.spec.ts:158]: Should group this with the other non-webhook tests. Maybe move the webhook validation tests to a separate spec?

**Pull request merged**